src/main/java/io/github/cottonmc/resources/CommonResources.java

```diff
@@ -46,18 +47,18 @@ public static void initialize() {
         builtinItem("gold", "gear", "plate", "dust");
 
         //These might get rods or molten capsules. They'd just need to be added to the end.
-        builtinRadioactive("uranium", BlockSuppliers.DIAMOND_TIER_ORE);
-        builtinRadioactive("plutonium", null);
-        builtinRadioactive("thorium", null);
+        builtinRadioactive("uranium", BlockSuppliers.DIAMOND_TIER_ORE, "gear", "plate", "ingot", "nugget");
+        builtinRadioactive("plutonium", null, "gear", "plate", "ingot", "nugget");
+        builtinRadioactive("thorium", null, "gear", "plate", "ingot", "nugget");
 
-        builtinItem("diamond", "gear", "dust");
-        builtinItem("emerald", "gear", "dust");
+        builtinItem("diamond", "gear", "plate", "dust");
+        builtinItem("emerald", "gear", "plate", "dust");
 
-        builtinGem("ruby", BlockSuppliers.IRON_TIER_ORE);
-        builtinGem("topaz", BlockSuppliers.IRON_TIER_ORE);
-        builtinGem("amethyst", BlockSuppliers.IRON_TIER_ORE);
-        builtinGem("peridot", BlockSuppliers.IRON_TIER_ORE);
-        builtinGem("sapphire", BlockSuppliers.IRON_TIER_ORE);
+        builtinGem("ruby", BlockSuppliers.IRON_TIER_ORE, MACHINE_AFFIXES);
```

Comment:
making gears or plates out of a gemstone is a bad idea.

Reply:
was told to generate json for everything even if it does not make sense, I think the idea is that we will just not have them in by default later, but as of right now with the cascade failure issues was told to register everything